x-pack/auditbeat/cache/cache.go

```diff
@@ -0,0 +1,59 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cache
+
+// Cache is just a map being used as a cache.
+type Cache struct {
```

Comment:
I wonder if the Cache doesn't need a mutex? Is it always called only from a single go-routine? If yes, then it's fine.

Reply:
Fetch is always called from the same goroutine. But I don’t see much of a downside to make it thread safe now.